LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -1,4 +1,3 @@
-<script src="../../../libs/js/promise-pollyfill/polyfill.min.js"></script>
```

Comment:
This is in templates\orgChart_import.php and LEAF_import_data.tpl as well.

Reply:
Going to leave those for a future IE9 "decruft" task